sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/csv/CSVDataSourceV2.scala

```diff
@@ -35,9 +38,12 @@ class CSVDataSourceV2 extends FileDataSourceV2 {
     CSVTable(tableName, sparkSession, options, paths, None, fallbackFileFormat)
   }
 
-  override def getTable(options: CaseInsensitiveStringMap, schema: StructType): Table = {
-    val paths = getPaths(options)
+  override def getTable(
+      schema: StructType,
+      partitions: Array[Transform],
+      properties: util.Map[String, String]): Table = {
+    val paths = getPaths(properties)
     val tableName = getTableName(paths)
-    CSVTable(tableName, sparkSession, options, paths, Some(schema), fallbackFileFormat)
+    CSVTable(tableName, sparkSession, properties, paths, Some(schema), fallbackFileFormat)
```

Comment:
`partitions` is ignored in all of these implementations. That looks like a bug to me.

Reply:
We shouldn't make this PR bigger and supports partitioning in file source. For now let's explicitly say that file source doesn't support user-specified partitioning: https://github.com/apache/spark/pull/25651/files#diff-3d5fde6e98a2856ba1b00ce6f172c4a8R72